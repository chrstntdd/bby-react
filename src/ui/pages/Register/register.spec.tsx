import React from "react";
import { shallow } from "enzyme";

import { Register } from "./";

window.scroll = jest.fn();

describe("<Register/>", () => {
  it("should render without crashing", () => {
    const wrapper = shallow(<Register />);

    expect(wrapper).toMatchSnapshot();
  });
});
