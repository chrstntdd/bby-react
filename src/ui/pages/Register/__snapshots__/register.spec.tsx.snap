// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Register/> should render without crashing 1`] = `
<main
  id="register-wrapper"
>
  <div
    className="show bg-white-darkest mx-auto shadow-lg rounded-b overflow-hidden flex flex-col items-center justify-center"
    id="register-card"
  >
    <Connect(RegisterForm)
      className="bg-white-darkest mx-auto shadow-lg rounded-b overflow-hidden flex flex-col items-center justify-center"
      id="sign-up"
    />
    <Link
      className="my-4 blue-accent hover:text-bby-blue no-underline"
      replace={false}
      to="/sign-in"
    >
      Sign In
    </Link>
  </div>
</main>
`;
